README.fr-fr.md

```diff
@@ -0,0 +1,227 @@
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/neon-nyan/CollapseLauncher-Page/main/images/NewBannerv2_color.webp"/>
+</p>
+
+##### Crédits pour l'arrière-plan Nahida de GI : [Rafa sur Pixiv](https://www.pixiv.net/en/users/3970196)
+
+[![en](https://img.shields.io/badge/README-en-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.md) [![jp](https://img.shields.io/badge/README-jp-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.ja-jp.md) [![id](https://img.shields.io/badge/README-id-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.id-id.md) [![pt](https://img.shields.io/badge/README-pt-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.pt-pt.md)
```

Comment:
Hi! 
You should change "README" to "lang" in this badges.

Reply:
```suggestion
[![en](https://img.shields.io/badge/lang-en-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.md) [![jp](https://img.shields.io/badge/lang-jp-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.ja-jp.md) [![id](https://img.shields.io/badge/lang-id-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.id-id.md) [![pt](https://img.shields.io/badge/lang-pt-red.svg)](https://github.com/neon-nyan/Collapse/blob/main/README.pt-pt.md)
```